### DISCUSSION TITLE
面向对象程序设计 - HAUE-CS-WIKI

### DISCUSSION THREAD

**Discussion by giscus** [Doge2077/haue-cs-wiki#22] [General]
# 面向对象程序设计 - HAUE-CS-WIKI

https://hauecs.wiki/%E7%BC%96%E7%A8%8B%E8%AF%AD%E8%A8%80%E5%9F%BA%E7%A1%80/C%2B%2B/%E9%9D%A2%E5%90%91%E5%AF%B9%E8%B1%A1%E7%A8%8B%E5%BA%8F%E8%AE%BE%E8%AE%A1/

**Comment by Meteor-Z** [comment]
这个黑马程序员不是很行，讲的不是modern c++,很多东西都没有讲

**Reply by Doge2077** [reply]
瘸子里面挑将军，如果以后有好的教程会更换的（

**Reply by Meteor-Z** [reply]
我觉得可以通过做cpp quiz来练习c++（bushi
